s3/pkg/meta/types/object.go

```diff
@@ -17,13 +17,13 @@ import (
 
 	"github.com/opensds/multi-cloud/s3/pkg/helper"
 	"github.com/opensds/multi-cloud/s3/pkg/meta/util"
-	"github.com/xxtea/xxtea-go/xxtea"
 	pb "github.com/opensds/multi-cloud/s3/proto"
+	"github.com/xxtea/xxtea-go/xxtea"
 )
 
 type Object struct {
 	*pb.Object
-	Rowkey           []byte // Rowkey cache
+	Rowkey []byte // Rowkey cache
```

Comment:
sorry,  what is Rowkey cache ?

Reply:
Sorry, Rowkey is abandoned, it's removed now.